lib/rules/require-newline-before-block-statements.js

```diff
@@ -97,6 +97,7 @@ module.exports.prototype = {
             errors.assert.differentLine({
                 token: prevToken,
                 nextToken: openingBrace,
+                allowExtra: true,
```

Comment:
Ye. https://github.com/jscs-dev/node-jscs/blob/master/test/rules/require-newline-before-block-statements.js#L30 
You're right. ;-) Nothing to do already.